apps/web/src/components/HoverCard/HoverCardCommunityInner.tsx

```diff
@@ -175,16 +184,20 @@ const StyledLink = styled(Link)`
   min-width: 0;
 `;
 
+const Section = styled(VStack)`
+  max-width: 250px;
+`;
```

Comment:
in `HoverCardUesrnameInner`, the width is set to `100%`, but here it's set to 250px. any reason for that difference?

want to be wary of these components diverging!